src/adapters/leveldb/index.js

```diff
@@ -571,11 +570,20 @@ function LevelPouch(opts, callback) {
       }, callback);
     }
 
+    function autoCompact(callback) {
+      var revsMap = new collections.Map();
+      fetchedDocs.forEach(function (metadata, docId) {
+        revsMap.set(docId, compactTree(metadata));
+      });
+      compact(revsMap, callback);
+    }
+
     function finish() {
       if (api.auto_compaction) {
         return autoCompact(complete);
+      } else {
```

Comment:
Small nit, everywhere else in the code has 2 space indents and we would like to be consistent